x-pack/plugins/ml/public/application/components/anomalies_table/anomaly_details.tsx

```diff
@@ -341,54 +325,94 @@ export class AnomalyDetails extends Component {
               />
             </h4>
           </EuiText>
-          <EuiDescriptionList
-            type="column"
-            listItems={listItems}
-            className="anomaly-description-list"
-          />
+
+          {listItems.map(({ title, description }) => (
+            <>
+              <EuiFlexGroup gutterSize="none">
+                <EuiFlexItem style={{ width: '180px' }} grow={false}>
+                  {title}
+                </EuiFlexItem>
+                <EuiFlexItem>{description}</EuiFlexItem>
+              </EuiFlexGroup>
+              <EuiSpacer size="xs" />
+            </>
+          ))}
           {othersCount > 0 && (
-            <EuiLink onClick={() => this.toggleAllInfluencers()}>
+            <EuiLink onClick={() => toggleAllInfluencers()}>
               <FormattedMessage
                 id="xpack.ml.anomaliesTable.anomalyDetails.anomalyDescriptionListMoreLinkText"
                 defaultMessage="and {othersCount} more"
                 values={{ othersCount }}
               />
             </EuiLink>
           )}
-          {numToDisplay > this.props.influencersLimit + 1 && (
-            <EuiLink onClick={() => this.toggleAllInfluencers()}>
+          {numToDisplay > influencersLimit + 1 && (
+            <EuiLink onClick={() => toggleAllInfluencers()}>
               <FormattedMessage
                 id="xpack.ml.anomaliesTable.anomalyDetails.anomalyDescriptionShowLessLinkText"
                 defaultMessage="Show less"
               />
             </EuiLink>
           )}
-        </React.Fragment>
+        </>
       );
     }
-  }
+  };
 
-  render() {
-    const { tabIndex } = this.props;
+  const renderContents = () => {
+    return (
+      <EuiFlexGroup>
+        <EuiFlexItem>
+          <div className="ml-anomalies-table-details" data-test-subj="mlAnomaliesListRowDetails">
+            {renderDescription()}
+            <EuiSpacer size="m" />
 
-    if (this.tabs !== undefined) {
-      return (
-        <EuiTabbedContent
-          tabs={this.tabs}
-          size="s"
-          initialSelectedTab={this.tabs[tabIndex]}
-          onTabClick={() => {}}
-        />
-      );
-    } else {
-      return (
-        <div className="ml-anomalies-table-details" data-test-subj="mlAnomaliesListRowDetails">
-          {this.renderDescription()}
-          <EuiSpacer size="m" />
-          {this.renderDetails()}
-          {this.renderInfluencers()}
-        </div>
-      );
-    }
+            <EuiFlexGroup gutterSize="l">
+              <EuiFlexItem css={dividerStyle}>
+                {renderDetails()}
+                {renderInfluencers()}
```

Comment:
Can you please either wrap `renderDetails` and `renderInfluencers` with `useCallback` or even better create a react component for it?

Reply:
Removed all render functions in https://github.com/elastic/kibana/pull/142999/commits/8be0cd78896ddcf4481a4ea71497ba008886a7a7